### ISSUE TITLE
Strava synchronization

### ISSUE THREAD

**Issue by Jetamias** [GoldenCheetah/GoldenCheetah#2256]
The activities upload to Strava must to synchronize whit the activities of the application, I mean, that they download the activities of Strava.

**Comment by Joern-R** [comment]
According to my investigatio the Strava API does not allow to download the original file loaded into Strava. The data downloadable through their APIs consists out of multiple parts which would need to be merged again to be considered as a single activity. Based on the API description it's not even 100% clear if all original data could be re-constructed.

The "Synchronize" feature of GoldenCheetah currently assume a single source (file) from the counterpart to be downloaded.

Unless you find a way to download your data from Strava as a "normal" activity file - ideally the original file you have uploaded, adding such functionality will most likely not happen (soon).

**Comment by liversedge** [comment]
The Streams API supports downloading raw data. I have opened a new ISSUE for this and will start working on it once the CloudService API is done.

See: #2442
